### PR TITLE
Turkish translation of "Why Do We Write super(props)?"

### PR DESCRIPTION
**Pull request by yusufff**
Hi Dan, this is Turkish translation of your first blog post.
Can we come up with an i18n solution? Like subpages `/tr` `/ru`?

### CODE REVIEW

**Comment by gaearon** [comment]
>Can we come up with an i18n solution? Like subpages /tr /ru?

Can you look into this? There must be some ready solution for Gatsby.

**Comment by gillkyle** [comment]
This Twitter thread gives some input on internationalization with Gatsby: https://twitter.com/s_ibylle/status/1064859690574757888

Might be a helpful reference!

**Comment by yusufff** [comment]
@gillkyle thank you, looked into that. Decided to use [gatsby-plugin-i18n](https://github.com/angeloocana/gatsby-plugin-i18n). Didn't find any strong alternative, do you have something?

**Comment by igorkamyshev** [comment]
Hello. You can use our solution from #24 after merge

File: src/pages/neden-super-props-yaziyoruz.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  constructor(props) {
+    super(); // 😬 props'u göndermeyi unuttuk
+    console.log(props);      // ✅ {}
+    console.log(this.props); // 😬 undefined 
+  }
+  // ...
+}
+```
+
+It can be even more challenging to debug if this happens in some method that’s called *from* the constructor. **And that’s why I recommend always passing down `super(props)`, even though it isn’t strictly necessary:**
```
Comment (MilanBarande) [inline src/pages/neden-super-props-yaziyoruz.md]: Looks like you forgot to remove the original text here

**Comment by yusufff** [comment]
> Hello. You can use our solution from #24 after merge

That's what I was going for @igorkamyshev 🙂 So, let's merge this after yours merged.

**Comment by sompylasar** [comment]
@yusufff Should the filename match the original for the translations to be linked together automatically, like in #24, or they are planned to be referenced manually? 

> src/pages/neden-super-props-yaziyoruz.tr.md

versus

> src/pages/why-do-we-write-super-props.ru.md

If the URLs need to be translated, there may be a way in Gatsby to somehow untangle the URLs from the filenames. But I guess if i18n plugin makes URLs with language prefixes, I wonder whether it would still make sense to localize URLs from SEO perspective, or content localization and URL language prefix are enough.

**Pull request merged**